### ISSUE TITLE
[Bug][Button]: Разобраться с доступностью loading кнопок

### ISSUE THREAD

**Issue by BlackySoul** [VKCOM/VKUI#8750]
### Описание

Наладить a11y кнопок со свойством `<Button loading />` – сейчас кнопка дополнительно становится `disabled` из-за чего слетает фокус.

### Версия

v7

**Comment by niktariy** [comment]
Здравствуйте! Почему-то не получилось скачать репозиторий.

Вот несколько рекомендаций:
- убрать зависимость атрибута `disabled` от `loading` , вместо него лучше задайте WAI-ARIA `aria-disabled={loading}`
- вижу вы используете `onClick={loading ? undefined : onClick}`, лучше будет превентить в обработчике
```js
const handleClick = (e) => {
   if (disabled || loading) {
    e.preventDefault();
    return;
  }
  // основной код
};
```
- можно добавить `aria-busy="true"` на кнопку или родительский элемент.

Пример:

```ts
<Tappable
  Component={restProps.href ? 'a' : 'button'}
  onClick={handleClick}
  disabled={disabled}
  aria-disabled={loading || disabled}
  aria-busy={loading}
  // остальные пропсы
>
  {loading && (
    <Spinner
      size="s"
      className={styles.spinner}
      disableAnimation={disableSpinnerAnimation}
      noColor
      aria-hidden="true"
    />
  )}
  <span className={styles.in}>
    ...
  </span>
</Tappable>

```

**Comment by inomdzhon** [comment]
Думаю нужно ещё в целом добавить отдельное свойство, которое бы позволяла делать доступное состояние `disabled`, вот референсы:

- https://ariakit.org/reference/button#accessiblewhendisabled
- https://base-ui.com/react/components/button

**Comment by vkcom-publisher** [comment]
✅ <a href="https://github.com/VKCOM/VKUI/releases/tag/v8.0.0" target="_blank">v8.0.0</a> 🎉
